lang/cz/maintenance.php

```diff
@@ -25,6 +25,12 @@
 		'button' => 'Generate!',
 		'success' => 'Successfully generated %d %s.',
 	],
+	'fill-filesize-sizevariants' => [
+		'title' => 'File sizes missing',
+		'description' => 'Found %d small variants without file size.',
+		'button' => 'Fetch data!',
+		'success' => 'Successfully computed sizes of %d small variants.',
```

Comment:
Is this only doing `small`?

Reply:
No, it is because size variants are not so clear for users outside of the code. For this reason, I preferred the nomenclature of _small variants_, which is already used in the diagnostics:
https://github.com/LycheeOrg/Lychee/blob/master/app/Actions/Diagnostics/Pipes/Checks/CountSizeVariantsCheck.php#L27

I don't mind changing though.

Reply:
Ah, ok. That works.